src/index.tsx

```diff
@@ -34,13 +34,14 @@ export type {
   AppStateObserver,
 } from './EditorApp'
 
-const ManuscriptEditor: React.FC<EditorAppProps> = ({
+const ManuscriptEditor: React.FC<EditorAppProps & { config: Config }> = ({
```

Comment:
Can we just add the new prop to `EditorAppProps`?

Reply:
I could find other options to clean this up, but I didn’t want to add `config` to `EditorAppProps` to avoid the need to pass it to every component that uses `EditorAppProps`, like `EditorApp` and `Main`.

Reply:
Got it.